docs/pages/world/resource-ids.mdx

```diff
@@ -0,0 +1,120 @@
+import { CollapseCode } from "../../components/CollapseCode";
+
+# Resource identifiers
+
+A `ResourceId` is a 32-byte value that uniquely identifies a resource in a `World`.
+It is two bytes of resource type followed by 14 bytes of namespace and then 16 bytes of the name of the actual resource.
+
+Currently, MUD supports these resource types:
+
+- [Table (`tb`)](https://github.com/latticexyz/mud/blob/main/packages/store/src/storeResourceTypes.sol#L12).
+  An onchain table whose information is available both onchain (through calls to `view` functions) and offchain (either through calls to `view` functions, events, or an [indexer](/services/indexer)).
+- [Offchain table (`ot`)](https://github.com/latticexyz/mud/blob/main/packages/store/src/storeResourceTypes.sol#L15).
+  An offchain table whose information is only available offchain (either through events, or through an [indexer](/services/indexer)).
+- [Namespace (`ns`)](https://github.com/latticexyz/mud/blob/main/packages/world/src/worldResourceTypes.sol#L13).
+  A namespace is a container for tables, offchain tables, and systems.
+  Namespaces are used for [access control](/world/namespaces-access-control).
+- [System (`sy`)](https://github.com/latticexyz/mud/blob/main/packages/world/src/worldResourceTypes.sol#L17).
+  A [system](/world/systems) contains logic that interact with table data onchain.
+
+## Creating resource identifiers
+
+To create a resource ID in your Solidity code you can use `WorldResourceIdLib.encode`.
+This function accepts a single parameter, a structure with three string fields:
+
+- `typeId` is the resource type.
+- `namespace` is the namespace in which the resource is located.
+  If this is the root namespace, use the empty string.
+- `name` is the name of the resource, except in the case of namespaces, where it is empty.
+
+For example, in the code below we create a resource for a table in the root namespace whose name is `Tasks`.
+
+<CollapseCode>
+
+```solidity filename="DeployKeyWithValueModule.s.sol" copy showLineNumbers {7-9,20-24}
```

Comment:
can we use a simper example that isn't keys with value?

Reply:
Consider it done.